### ISSUE TITLE
与林业（版本：forestry_1.12.2-5.8.2.380）共存产生错误，（建筑版本：buildcraft-all-7.99.21）

### ISSUE THREAD

**Issue by ZXANXUSZ** [BuildCraft/BuildCraft#4353]
在任意地图上，放置一个3*4*3的林业自动农场组件，（包含收货箱、齿轮箱、控制箱、水箱当中任意一个），会直接弹出游戏。

**Comment by bookerthegeek** [comment]
是否可以从游戏崩溃时获得崩溃报告？

**Comment by ZXANXUSZ** [comment]
[crash-2019-02-09_00.38.43-server.txt](https://github.com/BuildCraft/BuildCraft/files/2847449/crash-2019-02-09_00.38.43-server.txt)
这是错误报告，下面是一些相关信息。
Java Version: 1.8.0_51
minecraft version:1.12.2
forge version:1.12.2-14.23.5.2768

added mods:
bdlib-1.14.3.12-mc1.12.2.jar
buildcraft-all-7.99.21.jar
forestry_1.12.2-5.8.2.380.jar
NotEnoughIDs-1.5.4.3.jar      
Advanced+Solar+Panels-4.3.0.jar
ae2stuff-0.7.0.4-mc1.12.2.jar
appliedenergistics2-rv6-stable-6.jar
CookingForBlockheads_1.12.2-6.4.44.jar
ImprovedBackpacks-1.12.2-1.2.0.3.jar
industrialcraft-2-2.8.108-ex112.jar
InventoryTweaks-1.64+dev.145.jar
jei_1.12.2-4.14.4.264.jar
jeivillagers-1.12-1.0.2.jar
journeymap-1.12-5.4.9.jar
JustEnoughCharacters-1.12.0-3.3.0.jar

**Comment by bookerthegeek** [comment]
[Gist](https://gist.github.com/bookerthegeek/94b72f9d0f70dba376a2cac06e83ffcf) of the crash log.

这看起来像林业的错误，而不是建筑工艺。是否可以在您的林业农场和旁边的管道上获取舱口的屏幕截图？

此外，崩溃是在农场形成时发生的，还是仅在运行一段时间后发生？

Translated:
This looks like an error for forestry and not for build-craft. Would it be possible to get a screenshot of the hatch on your forestry farm and the pipe next to it please?

Also, is the crash happening as soon as the farm forms, or only when it is running for a while?

**Comment by AlexIIL** [comment]
I think that's https://github.com/ForestryMC/ForestryMC/issues/2340 which has been fixed in 
forestry_1.12.2-5.8.2.382.jar ?

**Comment by AlexIIL** [comment]
(Also thanks for helping :D )

**Comment by bookerthegeek** [comment]
Yep, same crash. I was on my phone so not able to google the crash. Also you're welcome.
